api/src/services/SolrIndexer.ts

```diff
@@ -134,6 +141,7 @@ class SolrIndexer {
                 }
             }
         }
+        fields.fedora_parent_id_str_mv = fedoraParents.map((parent) => parent.pid);
```

Comment:
what does this key mean? I usually avoid abbreviations since the meaning could be ambiguous to the someone maintaining the code. mv could mean move or multi-value, or market value.

Reply:
This is a Solr field name. The abbreviations are required, because they are built into the Solr schema.

Reply:
ok